en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-api-in-choreo-connect.md

```diff
@@ -26,40 +32,35 @@ Create a REST type API using one of the following methods:
 
 For this example, let's create an API via the WSO2 API Manager Publsiher Portal.
 
-### Step 3 - Deploy the API in API Manager
+### Step 2 - Deploy the API in API Manager
 
  The guide [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api) will explain how you could easily deploy the API you just created.
 
 That's it! To invoke the API follow the steps [here](#step-4-invoke-the-api).
 
-During the startup, Choreo Connect will check the `config.toml` to see if the `controlPlane` configuration has been enabled. If so, it will start fetching all the necessary artifacts that belongs to the gateway environment given in `environmentLabels`. These artifacts include deployed APIs, Applications, Subscriptions, Polices, information related to Key Managers, etc.
+!!! info
+
+    During the startup, Choreo Connect will check the `config.toml` to see if the `controlPlane` configuration has been enabled. If so, it will start fetching all the necessary artifacts that belongs to the gateway environment given in `environmentLabels`. These artifacts include deployed APIs, Applications, Subscriptions, Polices, information related to Key Managers, etc.
```

Comment:
```suggestion
    During the startup, Choreo Connect will check the `config.toml` to see if the `controlPlane` configuration has been enabled. If so, it will start fetching all the necessary artifacts that belong to the Gateway environment given in `environmentLabels`. These artifacts include deployed APIs, Applications, Subscriptions, Policies, information related to Key Managers, etc.
```